src/Avalonia.Diagnostics/Diagnostics/DevTools.cs

```diff
@@ -144,5 +144,23 @@ private static void DevToolsClosed(object? sender, EventArgs e)
             window.Closed -= DevToolsClosed;
             s_open.Remove((IDevToolsTopLevelGroup)window.Tag!);
         }
+
+        public static bool IsBelongsToDevTool(this Visual v)
```

Comment:
"IsBelongsToDevTool" doesn't sound right. It should be either BelongsToDevTool or DoesBelongToDevTool.
Also, I would put `internal` modifier just in case.